roles/confluent.variables_handlers/defaults/main.yml

```diff
@@ -85,24 +85,26 @@ zookeeper:
   jaas_file: /etc/kafka/zookeeper_jaas.conf
 
 kafka_broker_default_listeners:
-  external:
-    name: "{{ 'EXTERNAL' if kafka_broker_configure_additional_brokers|bool else 'DEFAULT' }}"
+  internal:
+    name: INTERNAL
     port: 9092
     ssl_enabled: "{{ ssl_enabled }}"
     ssl_mutual_auth_enabled: "{{ ssl_mutual_auth_enabled }}"
     sasl_protocol: "{{ sasl_protocol }}"
```

Comment:
can put an if statement around this sasl protocol so no one has to worry about the internal listener... maybe document the ssl_enabled, ssl_mutual_auth_enabled, and sasl_protocol vars for the interbroker listener, but component listener MUST NOT be touched... a better implementation would be to disregard their edits